zeppelin-server/src/test/java/org/apache/zeppelin/service/NotebookServiceTest.java

```diff
@@ -263,6 +264,22 @@ public void testNoteOperations() throws IOException {
     assertEquals(2, notesInfo.size());
     verify(callback).onSuccess(notesInfo, context);
 
+    // test moving corrupted note to trash
+    Note corruptedNote = notebookService.createNote("/folder_1/corruptedNote", "test", true, context, callback);
+    String corruptedNotePath = notebookDir.getAbsolutePath() + corruptedNote.getPath() + "_" + corruptedNote.getId() + ".zpln";
+    // corrupt note
+    try (FileWriter myWriter = new FileWriter(corruptedNotePath)) {
+      myWriter.write("{{{I'm corrupted;;;");
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+    notebookService.moveNoteToTrash(corruptedNote.getId(), context, callback);
+    reset(callback);
+    notesInfo = notebookService.listNotesInfo(false, context, callback);
+    assertEquals(3, notesInfo.size());
+    verify(callback).onSuccess(notesInfo, context);
+    notebookService.removeNote(corruptedNote.getId(), context, callback);
```

Comment:
Add test of removing note from trash

Reply:
Test of removing note from trash has been added into zeppelin-zengine/src/test/java/org/apache/zeppelin/notebook/NotebookTest.java, thx